webapp-django/crashstats/crashstats/utils.py

```diff
@@ -347,7 +347,7 @@ def ratelimit_rate(group, request):
     Otherwise return a number according to
     https://django-ratelimit.readthedocs.org/en/latest/rates.html#rates-chapter
     """
-    if group == 'crashstats.api.views.model_wrapper':
+    if group in ('crashstats.api.views.model_wrapper', 'crashstats.api.views.crash_verify'):
```

Comment:
I needed to do this otherwise `crash_verify` wouldn't work with rate limiting.

Having said that, I'm not positive I understand what this code is supposed to do. It's curious.